Contracts/TokenAuth Api/request.json

```diff
@@ -0,0 +1,7 @@
+//GET /api/tokenauth
+
+//Header
+{
+    //[Mandatory] Access Token to be validated
+    "AuthKey": "adba-sd123.244_gss34"
```

Comment:
1. update the name of url
2. make the parameter names consistent (headers are not exposed as json)